interface/fax/fax_dispatch.php

```diff
@@ -763,7 +764,7 @@ function allCheckboxes(issel) {
    </td>
   </tr>
   <tr>
-   <td class='itemtitle' nowrap><?php echo xlt('To{{Destination}}'); ?></td>
+   <td class='itemtitle' nowrap><?php echo xlt('To'); ?></td>
```

Comment:
revert the change in above line

Reply:
@bradymiller, not sure what this means.  I never changed this line.

Reply:
hi @vrjula , Your file was likely just a little outdated. Can fix this by just changing it to:
```
<td class='itemtitle' nowrap><?php echo xlt('To{{Destination}}'); ?></td>
```